packages/central-server/src/apiV2/taskComments/CreateTaskComment.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import { RECORDS } from '@tupaia/database';
+import { CreateHandler } from '../CreateHandler';
+import { assertAnyPermissions, assertBESAdminAccess } from '../../permissions';
+import { assertUserHasTaskPermissions } from '../tasks/assertTaskPermissions';
+
+/**
+ * Handles POST endpoints:
+ * - /tasks/:parentRecordId/taskComments
+ */
+
+export class CreateTaskComment extends CreateHandler {
+  parentRecordType = RECORDS.TASK;
+
+  async assertUserHasAccess() {
+    const createPermissionChecker = accessPolicy =>
+      assertUserHasTaskPermissions(accessPolicy, this.models, this.parentRecordId);
+
+    await this.assertPermissions(
+      assertAnyPermissions([assertBESAdminAccess, createPermissionChecker]),
+    );
+  }
+
+  async createRecord() {
+    const { id: userId } = this.req.user;
+    const user = await this.models.user.findById(userId); // Check if user exists
+    if (!user) {
+      throw new Error(`User with id ${userId} not found`);
+    }
+
+    const { full_name: userFullName } = user;
```

Comment:
Does this work? I would have thought you'd need to do something like:
```suggestion
    const userFullName = user.fullName;
```

Reply:
Yep it does work :) I made a custom column selector a while ago called `full_name` since we always seem to want to merge `first_name` and `last_name`, and so `full_name` is available as a property on the user now when grabbing from the model.